build.sbt

```diff
@@ -18,7 +18,10 @@ inThisBuild(
         "scm:git:git@github.com:gemini-hlsw/gpp-ui.git"
       )
     ),
-    scalaVersion := "2.13.1"
+    scalaVersion := "2.13.1",
+    scalacOptions ++= Seq(
+      "-Ymacro-annotations"
```

Comment:
isn't this included in sbt-gsp? Maybe it should

Reply:
Doesn't seem to be. Maybe it should.